LICENSE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 UAVCAN
+Copyright (c) 2020 Amazon
```

Comment:
Please use your own copyright (Incessant Gravity, right?). You are writing this code and releasing it under MIT. Amazon is simply supporting your development efforts within the terms of our contract. You'll see we specify the copyright in that contract is to be your own.

Reply:
I kept the previous Copyright and added my own.